anomalib/utils/loggers/comet.py

```diff
@@ -0,0 +1,116 @@
+
+    Args:
+        api_key: Required in online mode. API key, found on Comet.ml. If not given, this
+            will be loaded from the environment variable COMET_API_KEY or ~/.comet.config
+            if either exists.
+        save_dir: Required in offline mode. The path for the directory to save local
+            comet logs. If given, this also sets the directory for saving checkpoints.
+        project_name: Optional. Send your experiment to a specific project.
+            Otherwise will be sent to Uncategorized Experiments.
+            If the project name does not already exist, Comet.ml will create a new project.
+        rest_api_key: Optional. Rest API key found in Comet.ml settings.
+            This is used to determine version number
+        experiment_name: Optional. String representing the name for this particular experiment on Comet.ml.
+        experiment_key: Optional. If set, restores from existing experiment.
+        offline: If api_key and save_dir are both given, this determines whether
+            the experiment will be in online or offline mode. This is useful if you use
+            save_dir to control the checkpoints directory and have a ~/.comet.config
+            file but still want to run offline experiments.
+        prefix: A string to put at the beginning of metric keys.
+        \**kwargs: Additional arguments like `workspace`, `log_code`, etc. used by
+            :class:`CometExperiment` can be passed as keyword arguments in this logger.
+
+    Raises:
+        ModuleNotFoundError:
+            If required Comet package is not installed on the device.
+        MisconfigurationException:
+            If neither ``api_key`` nor ``save_dir`` are passed as arguments.
+    
+    Example:
+        >>> from anomalib.utils.loggers import AnomalibCometLogger
+        >>> from pytorch_lightning import Trainer
+        >>> comet_logger = AnomalibCometLogger()
+        >>> trainer = Trainer(logger=comet_logger)
+
+    See Also:
+        - `Demo in Google Colab <https://tinyurl.com/22phzw5s>`__
+        - `Comet Documentation <https://www.comet.com/docs/v2/integrations/ml-frameworks/pytorch-lightning/>`__
+    """
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        save_dir: Optional[str] = None,
+        project_name: Optional[str] = None,
+        rest_api_key: Optional[str] = None,
+        experiment_name: Optional[str] = None,
+        experiment_key: Optional[str] = None,
+        offline: bool = False,
+        prefix: str = "",
+        **kwargs
+    ) -> None:
+        super().__init__(
+            api_key=api_key,
+            save_dir=save_dir,
+            project_name=project_name,
+            rest_api_key=rest_api_key,
+            experiment_name=experiment_name,
+            experiment_key=experiment_key,
+            offline=offline,
+            prefix=prefix,
+            **kwargs
+        )
```

Comment:
Let's add framework tracking so we can follow how many experiment were using the Anomalib integration:
```suggestion
        )
        self.experiment.log_other("Created from", "Anomalib")
```